core/storage/suggestion/gae_models.py

```diff
@@ -681,7 +681,9 @@ def get_in_review_translation_suggestions_with_exp_ids_by_offset(
 
         Args:
             limit: int|None. Maximum number of entities to be returned. If None,
-                returns all matching entities.
+                returns all matching entities. This parameter is made optional
+                due to the sorting implementation required for sorting results
```

Comment:
Explain this more please -- your explanation should be clear about *why* the client-side implementation requires all entities to be returned. Assume the developer reading this has no idea about what's going on in the frontend and hasn't read your design doc.